presto-docs/src/main/sphinx/release/release-0.205.rst

```diff
@@ -0,0 +1,72 @@
+=============
+Release 0.205
+=============
+
+General Changes
+---------------
+
+* Fix parsing of row type signatures with spaces and types containing spaces.
+* Fix distributed planning failure for complex queries when using bucketed execution.
+* Fix a bug that triggers checks for ``SELECT`` privileges for tables in ``DELETE`` queries.
+  This bug only affects connectors that implement ``checkCanSelectFromColumns()``.
+* Fix a bug in enforcing column access controls when querying views. This bug
+  only affects connectors that implement the column access control methods.
+* Fix a bug in :func:`ST_ExteriorRing` to make it only work with polygons.
+* Add the ``task.min-drivers-per-task`` and ``task.max-drivers-per-task`` config options.
+  The former specifies the guaranteed minimum number of drivers a task will run concurrently
+  given that it has enough work to do. The latter specifies the maximum number of drivers
+  that can run concurrently for a task.
+* Add ``concurrent-lifespans-per-task`` config property to control the default value
+  of ``concurrent_lifespans_per_task`` session property.
+* Add ``query_max_total_memory`` session property and ``query.max-total-memory`` config
+  property. Queries will be aborted after their total (user + system) memory reservation
+  exceeds this threshold.
+* Improve the error message when access is denied when selecting from a view due to the
+  view owner having insufficient permissions to create the view.
+* Improve stats calculation for outer joins and correlated subqueries.
+* Change ``DELETE FROM <table> WHERE <condition>`` to check that the user has ``SELECT``
+  privileges on the ``WHERE`` condition, which is more in line with the ANSI standard.
```

Comment:
"more in line" is weak and seems to imply that we are not strictly following the standard. Should we change the second part to *"as is required by the SQL standard"*?

(it's now "SQL standard" rather than "ANSI", now that it's an ISO standard)